install/v2plugin/startcontiv.sh

```diff
@@ -77,9 +87,24 @@ if [ $plugin_role == "master" ]; then
         sleep 5
         echo "Restarting Netmaster " >> $BOOTUP_LOGFILE
     done &
+
+    echo "Waiting for netmaster to be ready for connections"
+    # wait till netmaster starts to listen
+    for i in $(seq 1 10); do
+        [ "$(curl -s -o /dev/null -w '%{http_code}' $control_url)" != "000" ] \
+           && break
+        sleep 1
+    done
+    if [ "$i" -ge "10" ]; then
+        echo "Failed to set forwarding mode, plugin will fail to enable"
+        exit 1
+    fi
+    sleep 1
+    echo "Netmaster ready for connections, setting forward mode to $fwd_mode"
+    /netctl --netmaster http://$control_url global set --fwd-mode "$fwd_mode"
```

Comment:
i think we need to check if fwd mode has been set or if it's allowed to update, or we can check the return code, if it's 5xx then retry, if it's 2xx or 4xx, it might be good to move on

Reply:
i did testing, the request is idempotent , as long as it's not changing the mode when there's network, we should be good